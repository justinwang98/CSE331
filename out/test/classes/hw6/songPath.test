# Test the path on a graph that contains several nodes

LoadGraph songs songs.tsv
ListNodes songs
FindPath songs JJLin AoiEir
