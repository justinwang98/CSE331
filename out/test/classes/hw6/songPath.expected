# Test the path on a graph that contains several nodes

loaded graph songs
songs contains: AoiEir BTS JJLin JayChou JokerXue Lisa Twice WangLeeHom
path from JJLin to AoiEir:
JJLin to Lisa via Chinese
Lisa to AoiEir via Japanese